pixart.js

```diff
@@ -1 +1,23 @@
+function changeBrushColor(){
+  var color = document.getElementById("color-field").value;
+  event.preventDefault();
+  document.querySelector("div.brush").style.background = color;
+};
 
+function changeSquareColor(){
+  var color = document.getElementById("color-field").value;
+  this.style.background = color;
```

Comment:
one small UI thing. In the current way this is coded, the color you paint in comes from the input box and not the background color of the brush